music_assistant/server/providers/tidal/__init__.py

```diff
@@ -172,16 +177,18 @@ async def get_config_entries(
     action: [optional] action key called from config entries UI.
     values: the (intermediate) raw values for config entries sent with the action.
     """
+    assert values is not None
```

Comment:
Not familiar enough with MA's take on config entries to know if this is sane. Further down it assumes values is not None without testing, and AFAICT if values was None in the next few chunks it would pass invalid data into `tidal_auth_url` and `tidal_pkce_login`. Without this restriction, we end up passing quality (and others) as a `str|None` instead of just `str`.

Reply:
yeah this makes sense